docs/pages/upgrading/automatic-agent-updates.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Set up Automatic Agent Updates
+description: Describes how to set up automatic agent updates.
+---
+
+We strongly recommend that Teleport users set up automatic agent updates to
```

Comment:
Similar to above, automatic upgrades is not a recommendation, it's a requirement.